mfr/extensions/tabular/libs/ezodf_tools.py

```diff
@@ -12,11 +12,13 @@ def ods_ezodf(fp):
     """
 
     workbook = ezodf.opendoc(fp.name)
-    sheet = workbook.sheets[0]
+    sheets = {}
 
-    list_data = [[cell.value for cell in row] for row in sheet.rows()]
+    for sheet in workbook.sheets:
+        list_data = [[cell.value for cell in row] for row in sheet.rows()]
 
-    header = header_population(list_data[0])
-    data = data_population(list_data)
+        header = header_population(list_data[0])
+        data = data_population(list_data)
+        sheets[str(sheet)] = (header, data)
 
     return header, data
```

Comment:
I understand that `header` and `data` refer to only the last sheet in the workbook - is that correct?